src/main/java/at/hannibal2/skyhanni/features/misc/ContributorManager.kt

```diff
@@ -16,7 +23,21 @@ object ContributorManager {
         contributors = event.getConstant<ContributorsJson>("Contributors").contributors.mapKeys { it.key.lowercase() }
     }
 
-    fun getTabListSuffix(username: String): String? = getContributor(username)?.suffix
+    @SubscribeEvent
+    fun onMobSpawnPlayer(event: MobEvent.Spawn.Player) {
```

Comment:
I dont think a spawn event should be used for such a cosmetic change, I think adding a new event for RenderEntityNameTag which allows overwriting the nametag would be better. This would also be more compatible with other mods